index.d.ts

```diff
@@ -1,71 +1,257 @@
-export type DataType = 'string';
+export interface WikidataResponse {
+	entities: {
+		[id: string]: Item;
+	};
+}
+
+export interface Item {
+	pageid: number;
+	ns: number;
+	title: string;
+	lastrevid: number;
+	modified: string;
+	type: string;
+	id: string;
+	labels: Labels;
+	descriptions: Descriptions;
+	aliases: Aliases;
+	claims: StatementMap;
+	sitelinks: Sitelinks;
+}
+
+export interface Labels {
+	[language: string]: LabelAndDiscription;
+}
+
+export interface Descriptions {
+	[language: string]: LabelAndDiscription;
+}
 
-export type DataValueType = 'string';
+export interface Aliases {
+	[language: string]: LabelAndDiscription[] | null;
+}
 
-export interface DataValue {
-	type: DataValueType;
+export interface LabelAndDiscription {
+	language: string;
 	value: string;
 }
 
+export interface StatementMap {
+	[propery: string]: (Statement)[] | null;
+}
+
+export type Snaks = CommonsMediaSnak
+| GlobeCoordinateSnak
+| MonolingualTextSnak
+| QuantitySnak
+| StringSnak
+| TimeSnak
+| URLSnak
+| WikibaseItemSnak
+| ExternalIdentifierSnak
+| GeoShapeSnak
+| MathSnak
+| TabularDataSnak
+| MusicalNotationSnak;
+
+export interface Statement {
+	mainsnak: Snaks;
+	type: "statement",
+	id?: string;
+	rank: "normal" | "deprecated" | "preferred";
+	qualifiers?: Qualifiers;
+	"qualifiers-order"?: (string)[] | null;
+	references?: (Reference)[] | null;
```

Comment:
I suspect that this null is not needed. because it can be undifined but not null